statsmodels/tsa/vector_ar/tests/test_var.py

```diff
@@ -639,6 +641,83 @@ def test_irf_trend():
     assert_allclose(irf_t.stderr()[1:4], irf.stderr()[1:4], rtol=0.03)
 
 
-if __name__ == '__main__':
-    import pytest
-    pytest.main([__file__, '-vvs', '-x', '--pdb'])
+class TestVARExtras(object):
+
+    @classmethod
+    def setup_class(cls):
+        mdata = sm.datasets.macrodata.load_pandas().data
+        mdata = mdata[['realgdp','realcons','realinv']]
+        data = mdata.values
+        data = np.diff(np.log(data), axis=0) * 400
+        cls.res0 = sm.tsa.VAR(data).fit(maxlags=2)
+
+    def test_process(self):
+        res0 = self.res0
+        k_ar = res0.k_ar
+        fc20 = res0.forecast(res0.endog[-k_ar:], 20)
```

Comment:
IIRC this should return a DataFrame.  Assert that?

Reply:
The unit tests that I added use only numpy arrays. So, in this test class there are no pandas objects.